src/Engine/GameEvents/HealUnitEvent.java

```diff
@@ -44,21 +44,22 @@ public GameEventQueue sendToListener(GameEventListener listener)
   public void performEvent(MapMaster gameMap)
   {
     if (null == payer)
-      unit.alterHealth(repairPowerHP, canOverheal);
-    else if( unit.isHurt() )
```

Comment:
Yeeted this check since the only thing it can do is make me fail to round up when I should sometimes.